terraform/modules/aws/coredns/templates/cloud-init.yaml

```diff
@@ -0,0 +1,65 @@
+#cloud-config
+
+write_files:
+  - path: /etc/coredns/Corefile
+    permissions: "0644"
+    owner: root
+    content: |
+      .:53 {
+        forward . 1.1.1.1 9.9.9.9
+        log
+        errors
+      }
+
+      firezone.internal:53 {
+        file /etc/coredns/db.firezone.internal
+        log
+        errors
+      }
+
+  - path: /etc/coredns/db.firezone.internal
+    permissions: "0644"
+    owner: root
+    content: |
+      $ORIGIN firezone.internal.
+      $TTL 1h
+      @   IN SOA   ns1.firezone.internal. admin.firezone.internal. (
+                      2024010501 ; Serial
+                      1h         ; Refresh (1 hour)
+                      10m        ; Retry (10 minutes)
+                      7d         ; Expire (7 days)
+                      1h         ; Minimum TTL (1 hour)
+                  )
+
+      httpbin     IN A     10.0.32.101
+      iperf       IN A     10.0.32.102
```

Comment:
Should we add the gateways here too? That'll allow us to test whether DNS works on gateways when trying to resolve itself (customer hit this issue this past weekend)

Reply:
Yep, no problem at all.  I actually just had a realization of how I can make it easier to add more records if needed.  Will push an update in just a minute.

Reply:
Added the gateway record as well as a record for the CoreDNS server itself.  Not sure if there is a need for the CoreDNS record, but figured it wouldn't hurt.